c/httpserver.c

```diff
@@ -5882,6 +5874,11 @@ void registerHttpServerModuleWithBase(HttpServer *server, STCBase *base)
 int mainHttpLoop(HttpServer *server){
   STCBase *base = server->base;
   /* server pointer will be copied/accessible from module->data */
+
+  /* Create semaphore table for datasets */
+  for(int i=0; i < N_SEM_TABLE_ENTRIES; i++)
```

Comment:
Why is this in the http server? This is service specific and an http server should not care about internal implementation of its services.